src/Hosting/Abstractions/src/MetricsPrototype/DefaultMetricsFactory.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Metrics;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Hosting.Abstractions.MetricsPrototype;
+
+internal class DefaultMetricsFactory : IMetricsFactory
+{
+    private readonly IOptions<MetricsOptions> _options;
+
+    public DefaultMetricsFactory(IOptions<MetricsOptions> options)
+    {
+        _options = options;
+    }
+
+    public Meter CreateMeter(string name)
+    {
+        // TODO: Configure meter with default tags.
+        return new Meter(name);
+    }
+
+    public Meter CreateMeter(MeterOptions options)
```

Comment:
I added a create method with options to set the name and version. Options allow for adding future arguments to creating a meter (if required)